src/main/java/com/spartronics4915/frc2020/subsystems/Launcher.java

```diff
@@ -0,0 +1,76 @@
+package com.spartronics4915.frc2020.subsystems;
+
+import com.spartronics4915.frc2020.Constants;
+import com.spartronics4915.lib.hardware.motors.SpartronicsSRX;
+import com.ctre.phoenix.motorcontrol.can.TalonSRX;
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.CANSparkMaxLowLevel.MotorType;
+import com.spartronics4915.lib.subsystems.SpartronicsSubsystem;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+public class Launcher extends SpartronicsSubsystem {
+    CANSparkMax mFlywheelMasterMotor;
+    CANSparkMax mFlywheelFollowerMotor;
+    TalonSRX mAngleAdjusterMotor;
+    TalonSRX mTurretMotor;
+    public Launcher() {
+        // Construct your hardware here
+        logInitialized(true);
+        //Two NEOs for flywheel (Master and follower, opposite directions)
+        mFlywheelMasterMotor = new CANSparkMax(Constants.kFlywheelMasterID,MotorType.kBrushless);
+        mFlywheelFollowerMotor = new CANSparkMax(Constants.kFlywheelFollowerID,MotorType.kBrushless);
+        mFlywheelFollowerMotor.follow(mFlywheelMasterMotor, true);
+        //One snowblower for angle adjustement
+        mAngleAdjusterMotor = new TalonSRX(Constants.kAngleAdjusterID);
+        //One BAG motor for turret
+        mTurretMotor = new TalonSRX(Constants.kTurretID);
+    }
+
+    // Outline your API here by creating specific methods.
+    // Each method should perform a _singular action_
+    // - eg. instead of a setIntake method, control each intake motor individually
+    // setIntake functionality should be implemented in a command.
+    public void turnToTarget() {
+        //rotates turret to face target
+    }
+
+    public void setAngle(double angle) {
+        //adjusts the angle to given angle
+    }
+
+    public void setRPM(double rpm) {
+        //begins spinning the flywheel at the given RPM
+    }
+
+    public double getAngle(double distance) {
```

Comment:
`getAngle` and `getRPM` are misleading names - it makes me think that they're returning the robot's current angle / RPM.

Reply:
`getRPM` is ok, IMO, because you'll always do `mLauncher.getRPM()`, so it's clear what the RPM is of.

`mLauncher.getAngle()` doesn't really make it clear if it's the hood or the turret that we're getting the angle of.

Reply:
@pietroglyph My understanding is that the intention of `getRPM` and `getAngle` is to _compute_ the RPM and angle needed given a distance. They'd be better named as such.

As far as the angle goes, last I heard from design the adjustable hood wasn't working out... if it still happens, we might not have it Week 1.